README.rst

```diff
@@ -1,6 +1,9 @@
 Flake8 Coding plugin
 =====================
 
+.. image:: https://img.shields.io/pypi/v/flake8-coding.svg
```

Comment:
PyPI badges are useful for adding a non-intrusive link to the package itself from Github, but if you want I'll remove...